ci-operator/config/backube/scribe/backube-scribe-master.yaml

```diff
@@ -1,37 +1,104 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.8"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
+  os:
+    name: centos
+    namespace: origin
+    tag: "8"
+  ubi_minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
 binary_build_commands: make all
 build_root:
   project_image:
     dockerfile_literal: |
       FROM registry.fedoraproject.org/fedora:33
       USER root
       RUN dnf install -y ShellCheck yamllint asciidoctor && \
-          dnf install -y make go diffutils && \
+          dnf install -y make go diffutils openssl && \
           dnf clean all && rm -rf /var/cache/dnf/*
+      RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+          chmod 700 get_helm.sh && \
+          ./get_helm.sh
       ENV GOPATH=/go GOCACHE=/go/src/.cache GO111MODULE=on
       RUN mkdir -p $GOCACHE /.cache $GOPATH/src/github.com/backube/testbin
       RUN chmod -R 777 /go /.cache
       RUN curl -sSLo envtest-bins.tar.gz "https://storage.googleapis.com/kubebuilder-tools/kubebuilder-tools-1.19.2-$(go env GOOS)-$(go env GOARCH).tar.gz"
       RUN tar -C $GOPATH/src/github.com/backube/testbin --strip-components=1 -zvxf envtest-bins.tar.gz
       WORKDIR $GOPATH/src/github.com/backube/scribe
+images:
+- dockerfile_literal: |
+    FROM src
+    COPY oc /usr/bin/oc
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: src-oc
+- context_dir: .
+  from: ubi_minimal
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.8
+  to: scribe
+- context_dir: mover-rclone
+  from: os
+  to: scribe-mover-rclone
+- context_dir: mover-restic
+  from: ubi_minimal
+  to: scribe-mover-restic
+- context_dir: mover-rsync
+  from: os
+  to: scribe-mover-rsync
+promotion:
```

Comment:
@JohnStrunk we don't _need_ this atm, but if you want to be able to pull from `registry.ci.openshift.org/scribe/scribe:latest`, etc, that's what this is for. Also, additional repository suites (microshift?) can use these images if they are promoted. ie, if microshift wants to test working w/ scribe, or any other suite. 
Also, this will all change if/when the scribe operator is added to operator hub.